### PR TITLE
Add privado as privacy management tool

### PR DESCRIPTION
**Pull request by hiteshbedre**
### New Submission Check List

Please ensure your submission ticks all of these boxes:

 - [x] This is Software as a Service not self hosted
 - [x] It has a free tier not just a free trial
 - [x] The submission mentions what is free
 - [x] The submission is not already present in the list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1612,6 +1612,7 @@ Table of Contents
   * [Iubenda](https://www.iubenda.com/) - Privacy and cookie policies along with consent management. Free tier offers limited privacy and cookie policy as well as cookie banners.
   * [Cookiefirst](https://cookiefirst.com/) - Cookie banners, auditing, and multi-language consent management solution. Free tier offers a one-time scan and a single banner.
   * [Ketch](https://www.ketch.com/) - Consent management and privacy framework tool. Free tier offers most features with a limited visitor count.
+  * [Privado](https://www.privado.ai/) - Tool identifies personal data as well as tracks the journey of processed data from the point of collection to defined sinks such as Third Parties, Databases, Logs, and APIs - thereby intelligently generating dataflows for your repository.
```
Comment (ripienaar) [inline README.md:1615]: Hard to see exactly but in the OSS version you run it locally/in CI and the output is JSON.  So is it really a SaaS? It also only support Java in the free version - worth mentioning assuming it is indeed a SaaS

Comment (hiteshbedre) [reply README.md:1615]: privado tool has a different product range. Their data mapping tool comes under saas offering and cli tool is a core component behind saas offering. Also, their saas offering supports more than 20 languages.

Comment (hiteshbedre) [reply README.md:1615]: Screenshot 2022-08-31 at 11 14 04 PM

The product range screenshot is attached here.

Comment (ripienaar) [reply README.md:1615]: Yes. But what is the free feature here exactly. To me it looks like the CLI mainly?

Comment (hiteshbedre) [reply README.md:1615]: yes, CLI is the free one.

Comment (ripienaar) [reply README.md:1615]: If it doesn’t interact with a SaaS then it’s not suited to the list - and from the description I think it doesn’t.

**Pull request closed without merging**